src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,52 @@
 package core.basesyntax;
 
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        List<String> lines = readLinesFromFile(fromFileName);
+        int supply = 0;
+        int buy = 0;
+
+        for (String line : lines) {
+            String[] parts = line.split(",");
+            if (parts[0].equals(SUPPLY)) {
+                supply += Integer.parseInt(parts[1]);
+            } else if (parts[0].equals(BUY)) {
```

Comment:
![image](https://github.com/mate-academy/jv-work-with-file/assets/156338484/6f7f9fbb-700a-40e0-b190-b25f78869d73)
Try this case